js/ui/scheduler/workspaces/ui.scheduler.work_space.js

```diff
@@ -3136,26 +3139,34 @@ const createDragBehaviorConfig = (
         }
     };
 
-    const onDragMove = () => {
-        if(isDefaultDraggingMode) {
-            return;
-        }
-
-        const MOUSE_IDENT = 10;
-
+    const getElementsFromPoint = () => {
         const appointmentWidth = getWidth(state.dragElement);
         const cellWidth = getCellWidth();
+
         const isWideAppointment = appointmentWidth > cellWidth;
+        const isNarrowAppointment = appointmentWidth <= DRAGGING_MOUSE_FAULT;
 
         const dragElementContainer = $(state.dragElement).parent();
         const boundingRect = getBoundingRect(dragElementContainer.get(0));
 
-        const newX = boundingRect.left + MOUSE_IDENT;
-        const newY = boundingRect.top + MOUSE_IDENT;
+        const newX = boundingRect.left;
+        const newY = boundingRect.top;
+
+
+        if(isWideAppointment) {
+            return domAdapter.elementsFromPoint(newX + DRAGGING_MOUSE_FAULT, newY + DRAGGING_MOUSE_FAULT);
+        } else if(isNarrowAppointment) {
+            return domAdapter.elementsFromPoint(newX, newY);
+        }
+        return domAdapter.elementsFromPoint(newX + appointmentWidth / 2, newY + DRAGGING_MOUSE_FAULT);
```

Comment:
Minor comment.

I prefer a switch in these cases, but it's a matter of taste:
```suggestion
        switch(true) {
          case isWideAppointment:
            return domAdapter.elementsFromPoint(newX + DRAGGING_MOUSE_FAULT, newY + DRAGGING_MOUSE_FAULT);
          case isNarrowAppointment:
            return domAdapter.elementsFromPoint(newX, newY);
          default:
            return domAdapter.elementsFromPoint(newX + appointmentWidth / 2, newY + DRAGGING_MOUSE_FAULT);
        }
```